.dockerignore

```diff
@@ -6,4 +6,5 @@ node-modules
 ledger
 .anchor
 docker/geyser-outputs
-docker/solana-outputs
\ No newline at end of file
+docker/solana-outputs
+snapshot/
```

Comment:
It might be wise to include .env as well

Reply:
agree. the `.env` option exists for local runs with cargo, but it should be irrelevant to docker, hence the changes to dotenvy error handling. we need not exit if .env wasn't found anywhere